init-exporter.go

```diff
@@ -165,22 +166,31 @@ func checkArguments() {
 	}
 }
 
+// configPath returns path to config
+func configPath() string {
+	if config_file := arg.GetS(ARG_CONFIG); config_file != "" {
```

Comment:
In this case I recomend to use [`fsutil.ProperPath`](https://godoc.org/pkg.re/essentialkaos/ek.v1/fsutil#ProperPath) method, which return first proper path to config file in slice. 

```go
func configPath() string {
	// Method return first path to readable, non-empty file from given slice
	return fsutil.ProperPath("FRS", []string{arg.GetS(ARG_CONFIG), DEFAULT_CONFIG_FILE})
}
```